lib/ansible/modules/windows/win_domain_computer.ps1

```diff
@@ -0,0 +1,218 @@
+#!powershell
+#
+# Copyright 2017, AMTEGA - Xunta de Galicia
```

Comment:
Please rewrite copyright statement and license using this shorter form:
```yaml
# Copyright: (c) 2017, AMTEGA - Xunta de Galicia
# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Reply:
done